test/__mocks__/text.mock.ts

```diff
@@ -256,3 +256,6 @@ export const UPDATE_MANAGE_TRUSTS_REVIEW_FORMER_BO_TITLE = "Review former benefi
 export const UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_FORMER_BO_TITLE = "Tell us about the former beneficial owner";
 export const UPDATE_MANAGE_TRUSTS_REVIEW_FORMER_BO_TABLE_HEADING = "<th scope=\"col\" class=\"govuk-table__header\">Name</th>";
 export const UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_LEGAL_ENTITY_TITLE = "Tell us about the legal entity";
+export const UPDATE_BANNER = "File an overseas entity update statement";
```

Comment:
I don't think these definitions are required - you can just reference the UPDATE_SERVICE_NAME and REMOVE_SERVICE_NAME constants in the main `config/index.ts` file. That's what's been done in the other tests.